pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -610,17 +645,13 @@ func listBundles(tx *gorm.DB, req *datastore.ListBundlesRequest) (*datastore.Lis
 	return resp, nil
 }
 
+// assumes non-nil node in request
```

Comment:
With this refactoring, it becomes clear that the the flow for each exported method is to 1) validate input, 2) take TX lock, 3) forward to unexported function to do the work. I don't think this comment is adding much value and risks becoming stale if the exported method does additional validation that would no longer be reflected in this comment.

I'd suggest removing it.

Reply:
will remove, but my overall concern was if other methods come in later to use this one